packages/plans-grid-next/src/components/features-grid/index.stories.tsx

```diff
@@ -78,10 +106,33 @@ export const NewsletterFlow: Story = {
 
 NewsletterFlow.storyName = '/setup/newsletter';
 
-export const CategorisedFeatures: Story = {
+export const TrailMapControl: Story = {
 	args: {
 		...PlansFlow.args,
+		trailMapVariant: 'control',
 		gridPlanForSpotlight: undefined,
+	},
+};
+
+export const TrailMapStructure: Story = {
+	args: {
+		...TrailMapControl.args,
+		trailMapVariant: 'treatment-structure',
+		enableCategorisedFeatures: true,
+		includePreviousPlanFeatures: true,
+	},
+};
+
+export const TrailMapCopy: Story = {
+	args: {
+		...TrailMapControl.args,
+		trailMapVariant: 'treatment-copy',
+	},
+};
+export const TrailMapCopyAndStructure: Story = {
+	args: {
+		...TrailMapControl.args,
+		trailMapVariant: 'treatment-copy-and-structure',
```

Comment:
Should `includePreviousPlanFeatures: true` be included here (as per the `TrailMapStructure` story)?

Reply:
The list of features returned for the copy & structure variant already includes the previous plan features, so we need to pass this prop as `true` only for the structure variant. This is also reflected in the value passed to the features grid in `client/my-sites/plans-features-main/index.tsx`:
<img width="788" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/9f6b7347-ea26-4072-a8db-8030995a6a30">